example/discord_bot/basic_prefix.py

```diff
@@ -0,0 +1,34 @@
+import discord
+
+import vacefron
+
+from discord.ext import commands
+
+intents= discord.Intents.default()
+
+
+class Bot(commands.Bot):
+    def __init__(self, **kwargs):
+        super().__init__(command_prefix="!", intents=intents, **kwargs)
+        self.vac_api= vacefron.Client()
```

Comment:
```suggestion
        self.vac_api = vacefron.Client()
```
Needs some space between the name and `=`

Reply:
This change hasn't been committed, why was this resolved?

Reply:
github doesn't allow me to commit because example folder is renamed, please re-review these again .